plugins/CorePluginsAdmin/Commands/InstallPlugin.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CorePluginsAdmin\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugin\Manager;
+use Piwik\Plugins\CorePluginsAdmin\PluginInstaller;
+use Piwik\Plugins\Marketplace\Marketplace;
+
+/**
+ * plugin:install console command.
+ */
+class InstallPlugin extends ConsoleCommand
+{
+    protected function configure(): void
+    {
+        $this->setName('plugin:install');
+        $this->setDescription('Install a plugin.');
+        $this->addOptionalArgument('plugin', 'The plugin name you want to install. Multiple plugin names can be specified separated by a space.', null, true);
+    }
+
+    protected function doExecute(): int
+    {
+        $input = $this->getInput();
+        $output = $this->getOutput();
+        $pluginManager = Manager::getInstance();
+
+        $pluginNames = $input->getArgument('plugin');
+
+        foreach ($pluginNames as $pluginName) {
+            if ($pluginManager->isPluginInstalled($pluginName)) {
+                $output->writeln(sprintf('<comment>The plugin %s is already installed.</comment>', $pluginName));
+                continue;
+            }
+
+            if (!$pluginManager->isPluginInFilesystem($pluginName) && !Marketplace::isMarketplaceEnabled()) {
+                $output->writeln(sprintf("<error>Marketplace is not enabled, can't install plugins.</error>"));
+                return self::FAILURE;
+            }
+            try {
+                $this->installPlugin($pluginName);
+                $output->writeln(sprintf("Installed plugin <info>%s</info>", $pluginName));
+            } catch (\Piwik\Plugins\CorePluginsAdmin\PluginInstallerException $e) {
+                $output->writeln(sprintf("<error>%s</error>", $e->getMessage()));
```

Comment:
In this case it might be good to also show a message like `Unable to install plugin XXXX: `.
Otherwise if you e.g. try to install multiple plugins and more than one plugin has an error it is not directly visible, which error message was for which plugin. Even though it's always in the provided order.

![image](https://github.com/user-attachments/assets/45e45248-afa1-44f8-aa0c-8bf7d38fe329)